docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[analyze-risk-score-data]]
+== View and analyze risk score data
+
+The {security-app} provides several options to monitor the change in the risk posture of hosts and users from your environment. We recommend that you prioritize alert triaging to identify anomalies or abnormal behavior patterns.
+
+Use the following places in the {security-app} to view and analyze risk score data:
+
+[discrete]
+=== Entity Analytics dashboard
+
+From the Entity Analytics dashboard, you can access entity key performance indicators (KPIs), risk scores, and levels. You can also click the number link in the **Alerts** column to investigate and analyze the alerts on the Alerts page.
+
+[role="screenshot"]
+image::images/ea-dashboard.png[Entity Analytics dashboard]
+
+[discrete]
+=== Alerts page
+
+Use the Alerts table to investigate and analyze host and user risk levels and scores.
+
+TIP: You can <<customize-the-alerts-table, customize the Alerts table>> by adding or reordering columns to easily visualize the relevant data.
+
+[role="screenshot"]
+image::images/alerts-table-rs.png[Risk scores in the Alerts table]
+
+You can use the drop-down filter controls to filter alerts by their risk score level. To do this, <<drop-down-filter-controls, edit the default controls>> to filter by `user.risk.calculated_level` or `host.risk.calculated_level`:
```

Comment:
Prefix the value here - Users can prioritize alert triage to address alerts associated with risky entities. 

We can then elaborate on how they can achieve this using drop-down filter controls.

Also, can we talk about this `prioritization` section first before `customizing alert table` ([GH ref](https://github.com/elastic/security-docs/pull/4167/files#diff-6093a7c3f674e20dc9597a819176f1d79e9ccdc54a7d7a28821a0f6096c00b45R19)) ?